build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Add a Google copyright notice at the top of this file. See other repo for an example.